<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server_dlg</class>
 <widget class="QDialog" name="server_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Server</string>
  </property>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>105</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>2</y>
     <width>405</width>
     <height>353</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="log_tedt"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Round:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Contest time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Current Question:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="round_line"/>
        </item>
        <item>
         <widget class="QLineEdit" name="time_line"/>
        </item>
        <item>
         <widget class="QLineEdit" name="question_line"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="change_btn">
          <property name="text">
           <string>&amp;Change Round</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="time_btn">
          <property name="text">
           <string>Set &amp;Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="question_btn">
          <property name="text">
           <string>Set &amp;Question</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="start_btn">
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="text">
         <string>S&amp;top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause_btn">
        <property name="text">
         <string>&amp;Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
